triagebot.toml

```diff
@@ -991,6 +991,10 @@ cc = ["@Zalathar"]
 [mentions."src/tools/opt-dist"]
 cc = ["@kobzol"]
 
+[mentions."tests/ui-fulldeps/lexer/unicode-version.run.stdout"]
+message = "Unicode version has changed."
```

Comment:
Suggestion: can you also duplicate the update reference instruction in the triagebot message, i.e.

> If the Unicode version changes are intentional, it should also be updated in the reference at <https://github.com/rust-lang/reference/blob/HEAD/src/identifiers.md>.

Reply:
Done.